x-pack/plugins/spaces/public/views/management/edit_space/enabled_features/toggle_all_features.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { EuiContextMenuItem, EuiContextMenuPanel, EuiLink, EuiPopover } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { capitalize } from 'lodash';
+import React, { Component } from 'react';
+interface Props {
+  onChange: (visible: boolean) => void;
+  disabled?: boolean;
+}
+
+interface State {
+  isPopoverOpen: boolean;
+}
+
+export class ToggleAllFeatures extends Component<Props, State> {
+  public state = {
+    isPopoverOpen: false,
+  };
+
+  public render() {
+    const button = (
+      <EuiLink onClick={this.onButtonClick} className={'spcFeatureChangeAllLink'}>
+        <FormattedMessage
+          id="xpack.spaces.management.toggleAllFeaturesLink"
+          defaultMessage="(change all)"
+        />
+      </EuiLink>
+    );
+
+    const items = (['show', 'hide'] as Array<'show' | 'hide'>).map(item => {
```

Comment:
Can you change the items to read "Show all", "Hide all"?

Reply:
You got it! Done in https://github.com/elastic/kibana/pull/34288/commits/3beb9ba0f751286f6b7c7f4cf8c3a89793ddd296